src/DataStructures/VariablesHelpers.hpp

```diff
@@ -11,12 +11,16 @@
 #include "DataStructures/Index.hpp"
 #include "DataStructures/SliceIterator.hpp"
 #include "DataStructures/Variables.hpp"
+#include "Utilities/Gsl.hpp"
 
 /*!
  * \ingroup DataStructures
  * \brief Slices the data within `vars` to a codimension 1 slice. The
- * slice has a constant grid coordinate in direction `sliced_dim`, and slices
- * the volume through the point `slice_point` of this coordinate.
+ * slice has a constant logical coordinate in direction `sliced_dim`,
+ * and slices the volume through the point `slice_point` of this
```

Comment:
I find "the point `slice_point` of this coordinate" to be confusing as you are not supposed to pass the value of the fixed coordinate, but instead the index that is fixed in `sliced_dim`.  

How about something like:
 "and slices the volume at `fixed_index` in `sliced_dim`.  For example, to get the lower boundary of `sliced_dim`,  pass `0` for `fixed_index`; to get the upper boundary, pass `extents[sliced_dim] - 1`"

Thoughts?